base/sparse/linalg.jl

```diff
@@ -532,10 +531,13 @@ function normestinv{T}(A::SparseMatrixCSC{T}, t::Integer = 2)
     end
```

Comment:
I had mistaken the deletion for an insertion. The issue was pre-existing then (i.e. I introduced it), thank you for fixing it.